README.md

```diff
@@ -81,6 +81,46 @@ end
 `text` fields will be full-text searchable. Other fields (e.g.,
 `integer` and `string`) can be used to scope queries.
 
+### Nested Documents support
```

Comment:
I think we need a reference (link) to a Solr documentation on nested documents here

Reply:
Agreed, will do, as well.